backend/contentBackend.js

```diff
@@ -36,54 +36,52 @@ async function authUserAndValidateRoute(jwt, route, invalidRouteError) {
  * into its valid HTML repr. Also returns frontmatter
  * attributes extracted from the original content.
  *
- * @param { string } jwt - token used for identification
+ * @param { string }  - token used for identification
```

Comment:
I think this is invalid JavaDoc (no param name), and AFAICT there is no matching parameter.